src/Templating/Helper/UploaderHelperInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Vich\UploaderBundle\Templating\Helper;
+
+interface UploaderHelperInterface
+{
+    /**
+     * Gets the public path for the file associated with the
+     * object.
+     *
+     * @param object|array $obj       The object
```

Comment:
This looks weird to me, I see there's no consistence in the codebase, but anyway I think that we should only allow objects

Reply:
@VincentLanglet what aout this?

Reply:
Sorry, I forgot about this ; it's fixed

Reply:
But then the third parameter is useless, isnt it ?